posts/2020-09-25-signed-certificate-with-acme-200010.adoc

```diff
@@ -83,6 +83,29 @@ For more information:
 * https://letsencrypt.org/[Popular public ACME CA]
 
 
+[#bugs]
+== Significant bugs fixed in this release
+
+We’ve spent some time fixing bugs. The following sections describe just some of the issues we resolved in this release. If you’re interested, here's the full list of link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A200010+label%3A%22release+bug%22+[fixed bugs in 20.0.0.10].
+
+=== Enabling openTracing with no tracer class configured
+
+With `MicroProfile-3.x` feature is enabled, `mpOpenTracing-1.3` is also enabled although users have no intention of using OpenTracing. Most likely the loading of a tracer will fail. Previously, mpOT code tried to load the tracer on every JAX-RS request. This issue has been fixed by loading the tracer only once. A dummy tracer is stored in the tracer map when the initial loading of the tracer is failed. Subsequent requests will not load the tracer again when the map has a dummy tracer. Also `AppStateListener` is added to listener when an application stopped. The corresponding entry in the tracer map will be removed. The `AppStateListener` also listens to `<Library />`` changes. All entries in the tracer map will be cleared.
+
+=== readiness check reports UP when application fails to start
+
+The mpHealth check reports UP when application fails to start. This bug was fixed by implementing a new test class, which now tests for:
```

Comment:
- The MicroProfile Health Check reports" or "The `mpHealth` feature reports".

Reply:
Done